code/ai/ai_flags.h

```diff
@@ -170,6 +170,7 @@ namespace AI {
 		Dont_limit_change_in_speed_due_to_physics_whack,
 		Guards_ignore_protected_attackers,
 		Standard_strafe_used_more,
+		Kamikaze_broadside_targets_subsystems,
```

Comment:
Minor, but why "broadside"?

Reply:
Because the bug only exists in `ai_cruiser_chase()`, which is only called by ships which attempt broadsides:
```C++
	if (sip->class_type > -1 && (Ship_types[sip->class_type].flags[Ship::Type_Info_Flags::AI_attempt_broadside])) {
		ai_cruiser_chase();
		return;
	}
```

Reply:
Ah, right, I always get `ai_cruiser_chase` and `ai_big_chase`, this is for big ships *doing* the kamikaze.